src/components/TopicMapItem/TopicMapItem.module.scss

```
@@ -15,6 +18,10 @@
 
     .inner {
       padding: 1em 1.25em;
+
+      &.withNote:not(.noImage) {
+        padding: calc(1em + $--note-btn-padding) 1.25em 1em;
```

Is the run time `calc` needed here? `$--note-btn-padding` is static and it's probably enough to do the calculation build time 
```suggestion
        padding: 1em + $--note-btn-padding 1.25em 1em;
```